care/facility/models/patient_consultation.py

```diff
@@ -74,6 +74,7 @@ class PatientConsultation(PatientBaseModel, PatientRelatedPermissionMixin):
     prescriptions = JSONField(default=dict)  # Deprecated
     procedure = JSONField(default=dict)
     suggestion = models.CharField(max_length=4, choices=SUGGESTION_CHOICES)
+    review_interval = models.IntegerField(default=-1)
```

Comment:
@vigneshhari Yep! It is. Whenever we receive a new daily log record, we update the review_time to set it to = "current time + review_interval". So that criteria is met.

Reply:
https://github.com/coronasafe/care/pull/1042/files#diff-523ea7aed3a17668736c27d6abb5b5d4462775f05b63043c46382390c608724dR125


https://github.com/coronasafe/care/pull/1042/files#diff-e126fcc15b8000c9011ba8d450d085f128e2d698d87e331912f1fa8c145f970eR155